cmd/auth/login.go

```diff
@@ -34,7 +34,47 @@ const minimalDbConnectVersion = "13.1"
 func newLoginCommand(persistentAuth *auth.PersistentAuth) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "login [HOST]",
-		Short: "Authenticate this machine",
+		Short: "Log into a Databricks workspace or account",
+		Long: `Log into a Databricks workspace or account.
+This command logs you into the Databricks workspace or account and saves
+the authentication configuration in a profile (in ~/.databrickscfg by default).
```

Comment:
FWIW, in Windows, I think this is something like `%USERPROFILE%\.databrickscfg`?

Reply:
Thanks, done! It'll show up as `(in %USERPROFILE%\.databrickscfg by default)` on windows.